submariner-operator/questions.yml

```
@@ -136,3 +136,9 @@ questions:
   group: "Advanced Configuration"
   description: "Cable driver implementation"
   label: "Cable Driver"
+- variable: submariner.healthcheckEnabled
+  type: boolean
+  default: true
+  group: "Advanced Configuration"
+  description: "Disable Healtcheck, if your CNI does not expose a pod IP on the nodes"
```

```suggestion
  description: "Disable Healthcheck"
```

As health check can be disabled for other reasons too, lets just use the term "Disable Healthcheck"